Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -65,22 +65,27 @@ the token returned by ``kubeadm init``
     name on each node.
 
 For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
-by using the following commands below. **Careful:** Be aware that this will break
-existing service connections. It will also stop service related traffic until the
-Cilium replacement has been installed:
+by using the following commands below.
+
+.. warning::
+   Be aware that removing ``kube-proxy`` will break existing service connections. It will also stop service related traffic
+   until the Cilium replacement has been installed:
 
 .. code-block:: shell-session
 
-    $ kubectl -n kube-system delete ds kube-proxy
-    $ # Delete the configmap as well to avoid kube-proxy being reinstalled during a Kubeadm upgrade (works only for K8s 1.19 and newer)
-    $ kubectl -n kube-system delete cm kube-proxy
-    $ # Run on each node with root permissions:
-    $ iptables-save | grep -v KUBE | iptables-restore
+   $ kubectl -n kube-system delete ds kube-proxy
+   $ # Delete the configmap as well to avoid kube-proxy being reinstalled during a Kubeadm upgrade (works only for K8s 1.19 and newer)
+   $ kubectl -n kube-system delete cm kube-proxy
+   $ # Run on each node with root permissions:
+   $ iptables-save | grep -v KUBE | iptables-restore
 
 .. include:: ../../installation/k8s-install-download-release.rst
 
-Next, generate the required YAML files and deploy them. **Important:** Make sure you correctly set your
-``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address and the kube-apiserver port number reported by ``kubeadm init`` (Kubeadm will use port ``6443`` by default).
+Next, generate the required YAML files and deploy them. 
+.. important::
+
+   Make sure you correctly set your ``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address
+   and the kube-apiserver port number reported by ``kubeadm init`` (Kubeadm will use port ``6443`` by default).
```

Comment:
Nit: We might still want to better wrap these lines (or the ones from the warning above).